packages/components/src/__actions__/Button/v3/_docs/Button.stories.tsx

```diff
@@ -18,81 +19,123 @@ export default meta
 
 type Story = StoryObj<typeof meta>
 
-export const Playground: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
+export const Playground: Story = {}
+
+export const ButtonVariants: Story = {
+  render: args => (
+    <>
+      <Button {...args} variant="primary" />
+      <Button {...args} variant="secondary" />
+      <Button {...args} variant="tertiary" />
+    </>
   ),
+  decorators: [
+    Story => (
+      <div className="flex gap-8">
+        <Story />
+      </div>
+    ),
+  ],
 }
 
-export const ButtonWithIcon: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
+export const ButtonVariantsReversed: Story = {
+  render: args => (
+    <ReversedColors isReversed={true}>
+      <Button {...args} variant="primary" />
+      <Button {...args} variant="secondary" />
+      <Button {...args} variant="tertiary" />
+    </ReversedColors>
   ),
-  args: {
-    children: (
-      <>
-        Label
-        <AddIcon role="presentation" />
-      </>
-    ),
+  parameters: {
+    reverseColors: true,
   },
+  decorators: [
```

Comment:
I don't think you're meant to need this as long as you've set the parameter? There should be a global decorator that should kick in

Reply:
Fixed the background with the unexpanded DocStory 👍